src/status_im2/contexts/communities/actions/accounts_selection/view.cljs

```diff
@@ -46,9 +52,37 @@
       {:community-name name
        :logo-uri       (get-in images [:thumbnail :uri])}]
      [gesture/scroll-view
-      [rn/view {:style style/content}
+      [:<>
        [quo/text
-        {:style               {:margin-top 24}
+        {:style               style/section-title
+         :accessibility-label :community-rules-title
+         :weight              :semi-bold
+         :size                :paragraph-1}
+        (i18n/label :t/address-to-share)]
+
+       [quo/category
+        {:list-type :settings
+         :data      [{:title             (i18n/label :t/join-as-a-member)
+                      :on-press          #(print "Show All Addresses Sheet")
```

Comment:
To keep the code consistent, we use `js/alert`. But even better would be `status-im2.common.not-implemented/alert`.